timemachine/md/barostat/utils.py

```diff
@@ -34,18 +35,18 @@ def get_bond_list(harmonic_bond_potential: HarmonicBond) -> List[Tuple[int, int]
         or if there are valence bonds not represented as harmonic bonds (e.g. as length constraints)
     """
 
-    bond_list = list(map(tuple, harmonic_bond_potential.get_idxs()))
+    bond_list = [(i, j) for i, j in harmonic_bond_potential.get_idxs()]
```

Comment:
Is this a mypy change?

Reply:
Yes, mypy isn't able to infer the type of the tuples. The error was:

> error: Incompatible return value type (got "List[Tuple[_T_co, ...]]", expected "List[Tuple[int, int]]")

This is fixed by making the tuple construction explicit.